backend/capellacollab/core/database/migration.py

```diff
@@ -99,3 +113,31 @@ def create_tools(db):
     tools.create_nature(db, papyrus.id, "UML 2.5")
     tools.create_nature(db, papyrus.id, "SysML 1.4")
     tools.create_nature(db, papyrus.id, "SysML 1.1")
+
+
+def create_t4c_instance_and_repositories(db):
+    LOGGER.info("Initialized T4C instance and repositories")
+    tool = tools.get_tool_by_name(db, "Capella")
+    version = tools.get_version_by_name(db, tool, "5.2.0")
+    default_instance = DatabaseT4CInstance(
+        name="default",
+        license="placeholder",
+        protocol=Protocol.tcp,
+        host="placeholder",
+        port=2036,
+        cdo_port=12036,
+        usage_api="http://placeholder.com",
+        rest_api="http://placeholder.com",
+        username="placeholder",
+        password="placeholder",
+        version=version,
```

Comment:
```suggestion
        name="default",
        license="placeholder",
        protocol=Protocol.tcp,
        host="localhost",
        port=2036,
        cdo_port=12036,
        usage_api="http://localhost:8086",
        rest_api="http://localhost:8081/api/v1.0",
        username="admin",
        password="password",
        version=version,
```

Some real values are nice for development.